Debugging.md

```diff
@@ -0,0 +1,83 @@
+
+### How can I get the logs of my game server?
+
+You can use the command `docker logs <nameOrHash>`. These logs contain everything your app sends to standard output/standard error streams. Bear in mind that these logs exist only for *existing* containers. If your game server crashes, our monitoring process in the VM will delete this container and create a new one.
+
+Consequently, it's a better practice to use GSDK to log from inside your game server. Check [here](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/integrating-game-servers-with-gsdk#logging-with-the-gsdk) for logging with GSDK and [here](https://docs.microsoft.com/en-us/gaming/playfab/features/multiplayer/servers/archiving-and-retrieving-multiplayer-server-logs) for accessing the logs from terminated game servers.
+
+### Can I connect "inside" a running container to see what's going on?
+
+Yup! You can do `docker exec -it <nameOrHash> powershell` on Windows and `docker exec -it <nameOrHash> bash` on Linux. There, you get access to a command line process in the container where you can issue native commands to debug/diagnose issues.
+
+> For this command to work on Linux, Bash shell must be installed in your base container. If it isn't, you can use Bourne shell by running `docker exec -it <nameOrHash> sh`.
+
+### How can I see the ports that are open on my container?
+
+Once you are connected "inside" the container, you can use `netstat -ano` on Windows and `netstat -tulpn` on Linux when you have a command line process inside your container (check the previous instruction).
+
+> If this command doesn't work on Linux, try installing netstat by `apt update && apt install net-tools`
+> On Linux, you can use `nestatst -tulpn` inside the VM to see that the ports are indeed open. Recall that VM ports will be different than container ports, you can see this port mapping with `docker ps` for your containers.
+
+## Linux specific instructions
+
+### On Linux, how can I see diagnostics about my container?
+
+You can install `apt install procps` and then run:
+
+- `ps -aux` for the current running processes (observer that your main process in the container has a PID of 1. If this process dies, your container will be gone)
+- `top` for real time process information
+
+### On Linux, how can I monitor TCP and UDP packets?
+
+You can use tcpdump utility. `apt update && apt install tcpdump` to install it. In order to use it for a specific port, you can do `tcpdump port 7777` for TCP and `tcpdump udp port 7778` for UDP. For more details, you can check [here](https://www.hugeserver.com/kb/install-use-tcpdump-capture-packets/).
+
+Worth mentioning is the fact that you can use tcpdump both from inside the VM and from inside the container. However, pay attention to the port you will be monitoring, since there is a port mapping between VM ports and container ports.
```

Comment:
>However, pay attention to the port you will be monitoring [](start = 109, length = 57)

Rephrase: However, ensure that the correct port value is used since ...